src/test/java/edu/usf/cutr/gtfsrtvalidator/test/feeds/combined/TripUpdateVehiclePostionAlertTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.usf.cutr.gtfsrtvalidator.test.feeds.combined;
+
+import edu.usf.cutr.gtfsrtvalidator.FeedMessageTest;
+import edu.usf.cutr.gtfsrtvalidator.validation.entity.LocationTypeReferenceValidator;
+import static junit.framework.TestCase.assertEquals;
+import org.junit.Test;
+
+
+public class TripUpdateVehiclePostionAlertTest extends FeedMessageTest{
+    
+    public TripUpdateVehiclePostionAlertTest() throws Exception{}
+    
+    @Test
+    public void testLocationTypeValidation(){
+        
+        LocationTypeReferenceValidator locationValidator = new LocationTypeReferenceValidator();
+        
+        stopTimeUpdate.setStopId("A");
```

Comment:
Please add a few more comments inline here for what each of these cases are testing for.